stable_baselines3/common/envs/identity_env.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, dim: Optional[int] = None, space: Optional[spaces.Space] = No
     def reset(self, *, seed: Optional[int] = None, options: Optional[Dict] = None) -> Tuple[T, Dict]:
         if seed is not None:
             super().reset(seed=seed)
+            self.action_space.seed(seed=seed + 1)
```

Comment:
Very important: the identity_env tests were flaky because of the lack of seeding here.